src/Spain/Mews.Fiscalizations.Spain.Tests/Basics.cs

```diff
@@ -56,12 +74,26 @@ public async Task CheckNif()
     [Retry(RetryCount)]
     public async Task PostInvoice()
```

Comment:
I know this is an old test, but how about we improve its naming? I am thinking something like "PostInvoice_WithValidData_Succeeds"